lamindb/models/_feature_manager.py

```diff
@@ -849,6 +849,13 @@ def add_values(
             values: A dictionary of keys (features) & values (labels, numbers, booleans).
             feature_field: The field of a reference registry to map keys of the dictionary.
             schema: Schema to validate against.
+
+        Examples:
+
+            To validate external features::
```

Comment:
This example is great but should likely be moved here:

<img width="782" height="689" alt="image" src="https://github.com/user-attachments/assets/6a1837bb-a19a-400d-8bdd-e71b9e790928" />

Or, alternatively, the stuff that's currently shown here https://docs.lamin.ai/lamindb.artifact#lamindb.Artifact.features should be moved here.